CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Fix time formatting for other cultures.
+- Fix MultipartBody serialization
```

Comment:
Can we move this up to a new release with a version bump in the csproj as well to release?